sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/TypedDocumentField.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.documentanalysis.models;
+
+import java.util.List;
+
+/**
+ * Strongly typed version of {@link DocumentField}
+ * @param <T> the type of value.
+ */
+public class TypedDocumentField <T> {
+    private T value;
+    private DocumentFieldType type;
+    private String content;
+    private List<BoundingRegion> boundingRegions;
+    private List<DocumentSpan> spans;
+    private Float confidence;
+
+    /**
+     * Get value of the field.
+     * @return the value of the field
+     */
+    public T getValue() {
+        return value;
+    }
+
+    private void setValue(T value) {
+        this.value = value;
+    }
+
+    /**
+     * Get the data type of the field value.
+     *
+     * @return the type value.
+     */
+    public DocumentFieldType getType() {
+        return this.type;
+    }
+
+    /**
+     * Get the field content.
+     *
+     * @return the content value.
+     */
+    public String getContent() {
+        return this.content;
+    }
```

Comment:
As a naive reviewer of this API, I don't know what the difference is between 'value' and 'content'. Can you add some JavaDoc to clarify the difference?

Reply:
```json
"valuePhoneNumber":"+19876543210","content":"987-654-3210"
```

Reply:
So, if the value is of type `Integer`, will content always be the string representation of the integer or will it be `null`?

Reply:
String representation of the integer value.
Example selection mark:
```json
valueSelectionMark":"unselected","content":":unselected:",
```